rotkehlchen/tests/unit/test_price_historian.py

```diff
@@ -28,6 +30,23 @@ def fixture_fake_price_historian(historical_price_oracles_order):
         data_directory=MagicMock(spec=Path),
         cryptocompare=MagicMock(spec=Cryptocompare),
         coingecko=MagicMock(spec=Coingecko),
+        manual_price_oracle=MagicMock(spec=ManualPriceOracle),
+    )
+    price_historian.set_oracles_order(historical_price_oracles_order)
+    return price_historian
+
+
+@pytest.fixture(name='manual_historian')
+def fixture_manual_historian(historical_price_oracles_order):
+    # NB: custom fixture for quick unit testing. Do not export.
+    # Since this is a singleton and we want it initialized everytime the fixture
+    # is called make sure its instance is always starting from scratch
+    PriceHistorian._PriceHistorian__instance = None
+    price_historian = PriceHistorian(
+        data_directory=MagicMock(spec=Path),
+        cryptocompare=MagicMock(spec=Cryptocompare),
+        coingecko=MagicMock(spec=Coingecko),
+        manual_price_oracle=ManualPriceOracle(),
```

Comment:
you don't really need this class. Since ManualPriceOracle has no external calls you don't need to mock it. So you can use it normally in the fake price historian above and get rid of this fixture.